lib/ansiblelint/utils.py

```diff
@@ -863,12 +878,10 @@ def get_playbooks_and_roles(options=None):
             playbooks.append(normpath(p))
             continue
 
-        if options.verbosity:
-            print('Unknown file type: %s' % normpath(p))
+        _logger.info('Unknown file type: %s', normpath(p))
 
-    if options.verbosity:
-        print('Found roles: ' + ' '.join(role_dirs))
-        print('Found playbooks: ' + ' '.join(playbooks))
+    _logger.info('Found roles: %s', ' '.join(role_dirs))
+    _logger.info('Found playbooks: %s', ' '.join(playbooks))
```

Comment:
should this be debug?

Reply:
Debug will not be displayed unless extra verbosity is asked and this commands are far too important to not be enabled by default (as we had several bugs around collection failures, like false-positives or false-negatives). I think is desirable to keep them like this, at least for the moment. In the long run, we may downgrade them to debug level. I would wait for a user to raise it as a request first, and later make a mini-survey.